config/streams.json

```diff
@@ -0,0 +1,5 @@
+[
+	"selfbots",
+	"bots",
+	"fire"
+]
```

Comment:
Not sure if the indentation is correct or not, but its not a big deal because it can be changed later on